controllers/networking-calico/pkg/charts/utils.go

```diff
@@ -54,7 +55,10 @@ func ComputeCalicoChartValues(network *extensionsv1alpha1.Network, config *calic
 	)
 
 	if config != nil {
-		calicoConfigValues["backend"] = config.Backend
+		switch config.Backend {
+		case calicov1alpha1.Bird, calicov1alpha1.None:
+			calicoConfigValues["backend"] = config.Backend
+		}
```

Comment:
This is only for validation correct?

Reply:
Yes, it is

Reply:
Those two validations should not be needed if we add two validating webhooks on Network but I felt that would be overkill?

Reply:
We are planning to support validating webhooks soon, and we should implement one for Calico as well. The configuration may be tampered by the end-users as its exposed in the `Shoot`, hence, we should prevent that some undesired values are sent to us. 
For now, I guess "this validation" is ok because we don't have the webhooks yet, but as soon as we have them we should implement it properly.